bedrock/mozorg/templates/mozorg/home/includes/survey.html

```diff
@@ -0,0 +1,50 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<style type="text/css">
+  #survey-message {
+    background: #fff0b3;
+    height: 30px;
+    line-height: 30px;
+    overflow: hidden;
+    top: -1px;
+    width: 100%;
+    text-align: center;
+    font-size: 12px;
+  }
+
+  #survey-message .survey-invite {
+    position: relative;
+    top: 1px;
+  }
+
+  #survey-message .survey-button {
+    background-color: #00b6e0;
+    border-radius: 2px;
+    color: #fff;
+    height: 20px;
+    left: 10px;
+    padding: 2px 5px;
+    position: relative;
+    top: 1px;
+  }
+
+  /* hide survey for smaller screens */
+  @media only screen and (max-width: 760px) {
+    #survey-message {
+      display: none;
+    }
+  }
+</style>
+
+<script>
```

Comment:
I know this totally works, but we currently have some CSP hopes and dreams for bedrock. We should probably try and avoid using inline `<style>` and `<script>` tags as policy? (even though it might be a pain for something so simple).